cmd/frontend/graphqlbackend/user.go

```diff
@@ -68,9 +68,10 @@ type UserResolver struct {
 
 // NewUserResolver returns a new UserResolver with given user object.
 func NewUserResolver(db database.DB, user *types.User) *UserResolver {
-	return &UserResolver{db: db, user: user, logger: log.Scoped("userResolver", "resolves a specific user").With(
-		log.Object("repo",
-			log.String("user", user.Username))),
+	return &UserResolver{
+		db: db, user: user, logger: log.Scoped("userResolver", "resolves a specific user").With(
+			log.Object("repo",
+				log.String("user", user.Username))),
```

Comment:
wow, that's not intendation, that's a massacre
```suggestion
		db: db,
		user: user,
		logger: log.Scoped("userResolver", "resolves a specific user").With(log.Object("repo", log.String("user", user.Username))),
```

Also: what is that `"repo"` doing here?

Reply:
I did not change anything here, this was go fmt doing its thing 🤷

Reply:
Yeah, sometimes you need to help it and add linebreaks :P